### PR TITLE
Kubernetes deployment option added

### PR DESCRIPTION
**Pull request by alparslanavci**

### CODE REVIEW

File: deploy/kubernetes/kubernetes.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+----
+
+== Containerizing the Application
+
+Firstly, please assure that Docker is running on your environment.
+
+In order to create the Docker image of the application, we will use the https://github.com/GoogleContainerTools/jib[Jib] tool, and its related Maven plugin. It allows to build containers from Java applications without a Docker file or even changing the pom.xml file. To build the image, run the command below:
+
+----
+mvn clean compile com.google.cloud.tools:jib-maven-plugin:2.1.0:dockerBuild
```
Comment (leszko) [inline deploy/kubernetes/kubernetes.adoc:36]: I would probably show the whole process with `Dockerfile`, JIB adoption is not so high and most people still use Dockerfiles, even for Spring Boot apps.

**Comment by mesutcelik** [comment]
@leszko My thinking was around like creating this guide and leaving user as much option as possible for the deployment part. So kubernetes is one part but docker/docker-compose/iaaS options would follow. 

the alternative is creating springboot-hazelcast-kubernetes but then we might end up creating similar for gcp/aws/azure/openshift/docker-compose...

I think it would be better if we can talk about those when htmls are published somewhere during the review process but that is not available. 
@hasancelik @alparslanavci are working on it.

**Comment by leszko** [comment]
> @leszko My thinking was around like creating this guide and leaving user as much option as possible for the deployment part. So kubernetes is one part but docker/docker-compose/iaaS options would follow.

I see the point, but as mentioned I think it makes the guide complex (to read and maintain). Anyway, we can try it. So apart from one more comment, it's +1 from my side.

**Comment by mesutcelik** [comment]
@leszko Should we merge this PR then you work on making `deploy/kubernetes/kubernetes.adoc` or we just need to close this PR and you send a new PR?

**Comment by leszko** [comment]
Let's maybe wait a moment, I'll look into that and check if this could be a separate guide with Kubernetes.

**Comment by leszko** [comment]
Closing this one as a similar guide is merged: https://github.com/hazelcast-guides/hazelcast-embedded-kubernetes/pull/1

**Pull request closed without merging**